### PR TITLE
create 20240308

### PR DESCRIPTION
**Pull request by gnsl318**
Create test_th.py 20240308

### CODE REVIEW

**Review by canboy123 — COMMENTED** [review]
why u put python code

File: test_th.py [hunk 1]
```diff
@@ -0,0 +1,4 @@
+import os
+
+
+print("Test th")
```
Comment (veryju) [inline test_th.py:4]: HI

File: test_th.py [hunk 2]
```diff
@@ -0,0 +1,4 @@
+import os
+
+
```
Comment (veryju) [inline test_th.py:3]: Good

File: test_th.py
Comment (canboy123) [inline test_th.py:1]: ahh... i see, you are adding new python file.

**Review by veryju — CHANGES_REQUESTED** [review]
I do not like you

**Review by canboy123 — COMMENTED** [review]
i just want to show comments on 1 file

File: test_th.py
Comment (canboy123) [inline test_th.py:1]: trial another comment

**Review by canboy123 — CHANGES_REQUESTED** [review]

**Pull request merged**